lib/pbench/server/api/resources/users_api.py

```diff
@@ -13,13 +13,16 @@
 
 
 class ForgotPassword(Resource):
-    def __init__(self, logger):
+    def __init__(self, config, logger, auth):
         self.logger = logger
         self.mail = Mail(app)
+        self.auth = auth
+        self.config = config
```

Comment:
It's kind of trivial (or worse), but it would be nice if these statements were ordered, somehow.  Alphabetical by attribute name would be nice; or, failing that, in the same order as the parameters in the signature.  (The latter is more "functional"; the former provides a better "reference" for the reader.)